src/main/java/com/google/sps/servlets/ListOrganizationsServlet.java

```diff
@@ -81,14 +81,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     QueryResultList<Entity> results = prepQuery.asQueryResultList(fetchOptions);
     ArrayList<Organization> requestedOrganizations = new ArrayList<Organization>();
 
-    /* Fills requestedOrganizations array with 4 fields of each org- name, phone, addres, and desc. */
+    /* Fills requestedOrganizations array*/
     for (Entity entity : results) {
-
-      Organization newOrg = new Organization((String) entity.getProperty("orgName"),
-                                             (String) entity.getProperty("orgEmail"),
+      long openingHour = (long) ((ArrayList)entity.getProperty("openHours")).get(0);
```

Comment:
robs comment: Is there a richer schema we can use for this? It feels error prone to have a convention that the second entry in "openHours" is the closing timestamp.

Reply:
We're planning on making a more detailed data structure for this that shows all of the open / closed times for each individual day of the week, but that is still a TODO()

Reply:
Agreed with andrews comment. Will add a TODO()

Reply:
As a question for the team: if we know we have to redo this part of the schema, what's the advantage of checking in this implementation vs leaving it out entirely as a simplification?

Reply:
We recognize that its an important aspect of the overall project (people being able to see when an organization is opened or closed) which is why we wanted to include a simpler implementation before we dive into a more complex implementation